src/code/z_camera.c

```diff
@@ -508,14 +508,14 @@ Vec3s* Camera_GetBgCamFuncDataUnderPlayer(Camera* camera, u16* bgCamCount) {
  * Returns the camera data index otherwise.
  */
 s32 Camera_GetWaterBoxBgCamIndex(Camera* camera, f32* waterY) {
-    PosRot playerPosShape;
+    PosRot playerPosRot;
```

Comment:
I think this is probably fine, but do think the old name is trying to convey that its the shape rotation (as opposed to world, like position). To not lose that information itd be like `playerWorldPosShapeRot` which is kinda long but yeah

it may be fine to sweep that detail under the rug and let it be implied, but just pointing it out

Reply:
Yeah I considered that `PosShape` could be a candidate to name these, but it would need to be at least `PosShapeRot` or `WorldPosShapeRot` which indeed would be too long

Note the current z_camera has mixed usage of PosRot and PosShape, I'm merely standardizing to PosRot which is the most used to fix the inconsistency (in particular there is a `camera->playerPosRot = playerPosShape;` in `Camera_InitDataUsingPlayer`)

Also overall camera uses the shape.rot more (it does use world.rot too in niche(?) cases), and it typically doesn't matter to use world or shape rot for player (besides zparallel/ztarget I guess), so I think it's fine